README.md

```diff
@@ -108,8 +107,8 @@ You can install the NodeObservability Operator by building and pushing the Opera
     EOF
     ```
 
-9. Create the `Subscription` object:
-    ```sh
+8. Create the `Subscription` object:
+    ```
     cat <<EOF | oc apply -f -
     apiVersion: operators.coreos.com/v1alpha1
     kind: Subscription
```

Comment:
@lmzuccarelli Is there any verification steps available post installation ? Please share